pkg/service/roommanager_service.go

```diff
@@ -53,7 +53,7 @@ func (s rtcRestService) Create(ctx context.Context, req *rpc.RTCRestCreateReques
 		return nil, ErrParticipantNotFound
 	}
 
-	if err := lp.HandleOffer(webrtc.SessionDescription{Type: webrtc.SDPTypeOffer, SDP: req.OfferSdp}); err != nil {
+	if err := lp.HandleOffer(webrtc.SessionDescription{Type: webrtc.SDPTypeOffer, SDP: req.OfferSdp}, 0); err != nil {
```

Comment:
Is the 0 correct here?

Reply:
Yeah, that is fine. That is the WHIP participant create path. WHIP does not have renegotiation